src/ashtech.h

```diff
@@ -121,7 +121,7 @@ class GPSDriverAshtech : public GPSBaseStationSupport
 	uint64_t _last_timestamp_time{0};
 
 	NMEADecodeState _decode_state{NMEADecodeState::uninit};
-	uint8_t _rx_buffer[ASHTECH_RECV_BUFFER_SIZE];
+	uint8_t _rx_buffer[ASHTECH_RECV_BUFFER_SIZE] {};
```

Comment:
This is one of the places I'd rather keep that way - It's just wasting cycles.

Reply:
Yes that's fine, this is just an experiment to see if we could have this enforced everywhere with a tool at the cost of occasional wasted effort. The current orb patterns are the main thing in the way that would actually be cost prohibitive.